sound/soc/intel/common/soc-acpi-intel-rpl-match.c

```diff
@@ -405,6 +405,11 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_rpl_machines[] = {
 					SND_SOC_ACPI_TPLG_INTEL_SSP_MSB |
 					SND_SOC_ACPI_TPLG_INTEL_DMIC_NUMBER,
 	},
+	{
+		.id = "INTC10B0",
+		.drv_name = "rpl_lt6911_hdmi_ssp",
+		.sof_tplg_filename = "sof-rpl-nocodec-hdmi-ssp02.tplg"
```

Comment:
Maybe remove "nocodec"? We use nocodec topologies for testing SSP with dummy codecs. But in this case, there is no SSP codec at all.

Reply:
Yes in this case there is no SSP codec, to indicate that only we added name as nocodec+hdmi-ssp02. we thought it will mislead if we don't mention it. Don't you want to mention it explicitly?

Reply:
We added the codec name in the topology name when it supports any specific codec. So, I assume no codec is supported by the topology if there is no codec name specified in the topology name. And, we have sof-adl-nocodec.tplg, sof-rpl-nocodec.tplg, etc for nocodec mode. We also added some postfix to indicate a variant, like sof-mtl-nocodec-ssp0-ssp2.tplg.
It is quite similar to sof-rpl-nocodec-hdmi-ssp02.tplg, but the "nocodec" means totally different things.
What do you think? @plbossart

Reply:
I don't like the use of nocodec either, but the cat is already out of the bag for previous platforms

````
soc-acpi-intel-adl-match.c:             .sof_tplg_filename = "sof-adl-nocodec-hdmi-ssp02.tplg"
````

so for consistency we probably want to follow the same (bad) convention...